.github/ISSUE_TEMPLATE/tck_challenge.md

```diff
@@ -0,0 +1,21 @@
+---
+name: TCK Challenge
+about: Create a TCK Challenge
+title: ''
+labels: 'challenge'
+assignees: ''
+
+---
+
+**Challenged tests**
+List the challenged tests with the fully qualified classnames and then the test methods, e.g.
+ee.jakarta.tck.faces.test.javaee7.multiFieldValidation.Spec1IT#testFailingPreconditionsNotAfterAllInputComponents
+
+**TCK Version**
+Specify the version of the TCK, e.g. Jakarta Faces 4.0.x
```

Comment:
This project is Jakarta Pages, not Jakarta Faces

Reply:
I took it as just an example... maybe clearer for the sake of each project to have a project specific example there, but that obviously takes more time for Emily to hunt down.

Reply:
I've made it page specific.